lib/matplotlib/backend_bases.py

```diff
@@ -2763,6 +2764,11 @@ def __init__(self, canvas):
         self.mode = ''  # a mode string for the status bar
         self.set_history_buttons()
 
+        @partial(canvas.mpl_connect, 'draw_event')
+        def define_home(event):
+            self.push_current()
+            canvas.mpl_disconnect(define_home)
```

Comment:
We need to update the docs on `mpl_disconnect` that this works with a function instead of a cid.

Independent of that,  I do not see where this is used.